iree/tools/run_mlir_main.cc

```diff
@@ -258,6 +265,22 @@ StatusOr<std::string> PrepareModule(
   return binary_contents;
 }
 
+// Returns a splitted input values from `filename` using comma as separater.
+std::vector<std::string> GetInputValues(const std::string& filename) {
+  std::string error_message;
+  auto file = mlir::openInputFile(filename, &error_message);
+  if (!file) {
+    NotFoundErrorBuilder(IREE_LOC)
```

Comment:
Make this file return StatusOr and return this instead

Reply:
Done